libraries/Microsoft.Bot.Builder.AI.Orchestrator/OrchestratorRecognizer.cs

```diff
@@ -82,11 +83,6 @@ public OrchestratorRecognizer(string modelFolder, string snapshotFile, ILabelRes
             InitializeModel();
         }
 
-        [JsonIgnore]
```

Comment:
I see that this property shipped with R13 at least. We cannot delete it without breaking backward compatibility.

Reply:
Will put it back.